plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/LibraryClasspathContainerInitializer.java

```diff
@@ -80,7 +81,11 @@ public void initialize(IPath containerPath, IJavaProject project) throws CoreExc
             new IClasspathContainer[] {container},
             new NullProgressMonitor());
       } else {
-        resolverService.resolveContainer(project, containerPath, new NullProgressMonitor());
+        IStatus result =
+            resolverService.resolveContainer(project, containerPath, new NullProgressMonitor());
+        if (!result.isOK()) {
+          throw new CoreException(result);
```

Comment:
the default message isn't very helpful. Could we provide a better custom message here?

Reply:
I've improved the message slightly; suggestions welcome.